tests/README.md

```diff
@@ -2,6 +2,24 @@
 
 When adding new features or making changes to the code, it's helpful to run a suite of tests to make sure various things are behaving as expected. This should not be necessary for users who are not changing the code at all, although it could still be nice to verify that the outputs users are seeing match those the developers expect for a few specific cases.
 
-To run the tests, copy `mpettest.py` to the repository root directory and run it. This will run a number of simulations and compare their outputs to those outputs from the downloaded reference outputs along with a few comparisons of simulations to their corresponding analytical results.
+To run the tests, execute ./bin/run\_locally.sh from the repository root. with
+the -h flag you can see which arguments are accepted.  This will run a number of
+simulations and store the results.
 
-Note that tests may "fail" even when things are okay, resulting from small numerical differences. If tests fail, it is helpful to look at the comparison plots generated by default within `mpet/tests/test_outputs/[time-stamped-directory]/plots` to see if the differences seem significant.
\ No newline at end of file
+To compare these outputs you can use pytest.
+
+```bash
+  pytest --baseDir=tests/ref_outputs/ --modDir=tests/test_outputs/20201208_154137/ tests/compare_tests.py
+```
+or you can compare specific tests:
+```bash
+  pytest --baseDir=tests/ref_outputs/ --modDir=tests/test_outputs/20201208_154137/ tests/compare_tests.py --tests test001 test002 --skip-analytic
+```
+or you can compare timings:
+```bash
+  pytest --baseDir=tests/ref_outputs/ --modDir=tests/test_outputs/20201208_154137/ tests/compare_timings.py --tests test001 test002 --skip-analytic
+```
+
+You can also compare different output folders, or against the reference solution.
+
+Note that tests may "fail" even when things are okay, resulting from small numerical differences. If tests fail, it is helpful to look at the comparison plots generated by default within `mpet/tests/test_outputs/[time-stamped-directory]/plots` to see if the differences seem significant.
```

Comment:
When I have encountered this problem I have usually used something like np.round(num, 3) in order round off the small numerical difference

Reply:
explained option for this in 2641c7e